File: Angular/src/app/models/customer.ts
export class Customer {
    customerId:number;
    customerName:string;
    phone:number;
    email:string;
    birthday:Date;

    constructor(customerId:number, customerName:string, phone:number, email:string, birthday:Date)
    {
        this.customerId=customerId;
        this.customerName=customerName;
        this.phone=phone;
        this.email=email;
        this.birthday=birthday;

    }
}

File: C#/Dal/Models/Customer.cs
using System;
using System.Collections.Generic;

namespace Dal_Repository.Models;

public partial class Customer
{
    public int CustomerId { get; set; }

    public string CustomerName { get; set; } = null!;

    public string Phone { get; set; }

    public string Email { get; set; } = null!;

    public DateTime Birthday { get; set; }

    public virtual ICollection<Purchase> Purchases { get; set; } = new List<Purchase>();
}
